app.js

```diff
@@ -6,68 +6,78 @@ const headEl = document.getElementById('head');
 const middleEl = document.getElementById('middle');
 const bottomEl = document.getElementById('bottom');
 const reportEl = document.getElementById('report');
-const catchphrasesEl = document.getElementById('catchphrases');
-const catchphraseInput = document.getElementById('catchphrase-input');
+const catchphraseEl = document.getElementById('catchphrases');
+const catchphraseInputEl = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
+let catchphrases = [];
 // set state for all of the character's catchphrases
 
 headDropdown.addEventListener('change', () => {
     // get the value of the head dropdown
-
+    const headValue = headDropdown.value;
     // increment the head change count state
-    
-    // update the dom for the head (use style.backgroundImage on the headEl div instead of trying to set the .src -- it's NOT an img tag!)
+    headCount++;
 
+    headEl.style.backgroundImage = `url(./assets/${headValue}-head.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
```

Comment:
You can update the stats by calling displayStats() here. Call this function in each dropdown eventlistener